src/extensions/extension.tsx

```diff
@@ -180,7 +178,8 @@ async function loadPageRecursive(type: RepoType, pageNum: number) {
 
   // If still have more results, recursively fetch next page
   console.log(`Parsed ${soFarResults}/${pageOfRepos.total_count} ${type}s`);
-  if (remainingResults > 0) return await loadPageRecursive(type, pageNum + 1); // There are more results. currentPage + 1 is the next page to fetch.
+  if (remainingResults > 0) return await loadPageRecursive(type, pageNum + 1);
+  // There are more results. currentPage + 1 is the next page to fetch.
```

Comment:
![image](https://user-images.githubusercontent.com/77577746/173076555-ccd8bb43-8282-41be-9cf5-28d76eaba917.png)
Because they are of the same position in a hierarchy so it having the same indent spaces as the lines around them, I'm not sure this is configurable.

Reply:
Done!